### PR TITLE
Docs entries for match messaging

### PR DESCRIPTION
**Pull request by uncleNight**
Hi! Some minor docs update and a section for match messaging (thought I should share because some points aren't exactly clear from Nakama docs for other clients, better avoid confusion).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -177,6 +178,38 @@ Listeners available:
...
+    dest_x = 1.0,
+    dest_y = 0.1,
+})
+
+local net_msg = {
+    match_data_send = {
+        match_id = match_id,     -- you get it from on_matchmakermatched() listener in case
+                                 -- of authoritative match, or after joining the relayed match;
+        op_code = 1,             -- pick the op_code for yourself depending on the gameplay event;
+        data = b64.encode(data), -- consider writing a convenience function for encoding your data
```
Comment (britzl) [inline README.md:201]: Perhaps this should be part of the nakama-defold extension?

**Comment by britzl** [comment]
Thanks! Great additions! I wonder if the json+b64 step could be either handled automatically or at least packaged to a utility function?

File: README.md (hunk 1 above)
Comment (uncleNight) [reply README.md:201]: Thought it'd be too daring to suggest... :D I'll add it as a util package if you don't mind.

**Comment by uncleNight** [comment]
I'd go with utility function first. The eventual goal would be to replace `send_socket()` calls with wrapper methods (like `send_match_message()`, `send_chat_message()` and so on (like what other Nakama clients have now), and for couple of these I already have the foundation in my game. So once it proves okay and tested, I'll contribute it back here if you don't mind, but sending a half-assed work right now would be simply impolite of me.

**Pull request merged**